src/oauth.test.ts

```diff
@@ -58,6 +66,7 @@ describe("OAuth", () => {
       ]);
       t.assert.deepStrictEqual(metadata.token_endpoint_auth_methods_supported, [
         "client_secret_post",
+        "client_secret_basic",
```

Comment:
Note: we explicitly don't have "none" here, even though it is technically supported, because we don't yet support public client registration.